internal/broker/broker.go

```diff
@@ -166,7 +166,7 @@ func (b *Broker) NewSession(username, lang, mode string) (sessionID, encryptionK
 	defer decorate.OnError(&err, "could not create new session for user %q", username)
 
 	sessionID = uuid.New().String()
-	session := sessionInfo{
+	session := session{
```

Comment:
I renamed the local variable to `s`, which should still be clear enough since this is the `NewSession` function.

Reply:
Well, you solved the issue in one function... But there's a lot of cases of the same issue throughout the file.

Reply:
The only other cases I see are function arguments named `session`. I don't see a problem with those, because it's clear that in those functions the `session` identifier refers to the argument, not the type.